src/hotspot/share/cds/filemap.cpp

```diff
@@ -1918,20 +1918,20 @@ void FileMapInfo::map_heap_regions_impl() {
   assert(is_aligned(relocated_closed_heap_region_bottom, HeapRegion::GrainBytes),
          "must be");
 
-  // Map the closed_archive_heap regions, GC does not write into the regions.
-  if (map_heap_data(&closed_archive_heap_ranges,
-                    MetaspaceShared::first_closed_archive_heap_region,
-                    MetaspaceShared::max_closed_archive_heap_region,
-                    &num_closed_archive_heap_ranges)) {
-    HeapShared::set_closed_archive_heap_region_mapped();
+  // Map the closed_heap regions, GC does not write into the regions.
```

Comment:
either add '_' between heap and regions or replace the "_' with space.

Reply:
I changed to `// Map the closed heap regions: GC does not write into these regions.` I also updated the comment for the open heap regions a few lines below.